docs/lib/QUICKSTART.md

```diff
@@ -422,26 +424,12 @@ When the uVisor is enabled, all NVIC APIs are rerouted to the corresponding uVis
 
 - The uVisor owns the interrupt vector table.
 - All ISRs are relocated to SRAM.
-- Code in a box can only change the state of an IRQ (enable it, change its priority, etc.) if the box registered that IRQ with uVisor at runtime, using the `NVIC_SetVector` API.
+- Code in a box can only change the state of an IRQ (enable it, change its priority, etc.) if the box registered that IRQ with uVisor via an IRQ ACL.
```

Comment:
- Please change "priority, etc." to "priority and so on" because we don't use Latin abbreviations in our docs. [Sorry - it looks like I missed this before.]
- Query: Could we replace "via" with "through" to replace the Latin word, or would that change the meaning?